### PR TITLE
fix(token): some token not showing, update blackdragon icon

### PR DESCRIPTION
**Pull request by superokid**
## Issues
fix Blackdragon, $Meta token not showing hidden by blacklisted token feature
update blackdragon icon
Screen Shot 2024-03-26 at 12 48 17

### CODE REVIEW

File: packages/e2e-tests/recover/manualImport.spec.ts
Comment (SteveKok) [inline packages/e2e-tests/recover/manualImport.spec.ts:1]: Please uncommit this and open a new PR.

Don't push commits that unrelated to the current PR title.

File: packages/frontend/src/redux/slices/tokens/index.js
Comment (SteveKok) [inline packages/frontend/src/redux/slices/tokens/index.js:1]: dollar sign `$` was filtered to make sure tokens such as `$META` won't be displayed.

Symbol length was set to `10` to make sure tokens name can't be meaningful enough for english meaning.

Please whitelist the blackdragon token by changing the following code:
```typescript
if (
    ['Native USDT', 'Native USDC', 'Bridged USDC', 'Bridged USDT'].includes(
        onChainFTMetadata.symbol
    )
) {
    return true;
}
```

**Review by SteveKok — CHANGES_REQUESTED** [review]
This PR is dangerous. It might unflag some tokens that originally flagged by us as malicious.

It is safer to whitelist tokens that we feel not a scam, instead of changing the rules.

File: packages/e2e-tests/recover/manualImport.spec.ts
Comment (superokid) [reply packages/e2e-tests/recover/manualImport.spec.ts:1]: fixing test every commit is better than hold into another branch

File: packages/frontend/src/redux/slices/tokens/index.js
Comment (superokid) [reply packages/frontend/src/redux/slices/tokens/index.js:1]: can show example of scam token that can pass the check?
how many token there will be hidden, & will need us to keep whitelist

**Review by SteveKok — APPROVED** [review]

**Pull request merged**